mods/ts/rules/aircraft.yaml

```diff
@@ -35,6 +35,52 @@ DPOD:
 		EmptySequence: pip-ammoempty
 		Palette: pips
 
+DPOD2:
```

Comment:
Why does this not replace the old DPOD? It seems that one will never be used. The only other use of drop-pods is as a scripted drop in two gdi campaign missions and I don't see a reason why those drop-pods would need to use different traits than the ones used in the support power.

Reply:
This is taken from the original rules.ini and I would like to keep those original actor names for compatibility.